src/transform/CollapseTable.js

```diff
@@ -13,48 +13,25 @@ const SECTION_TOGGLED_EVENT_TYPE = 'section-toggled'
  * @return {!Array<string>}
  */
 const getTableHeader = (element, pageTitle) => {
-  let thArray = []
-
-  if (!element.children) {
-    return thArray
-  }
-
-  for (let i = 0; i < element.children.length; i++) {
-    const el = element.children[i]
-
-    if (el.tagName === 'TH') {
-      // ok, we have a TH element!
-      // However, if it contains more than two links, then ignore it, because
-      // it will probably appear weird when rendered as plain text.
-      const aNodes = el.querySelectorAll('a')
-      // todo: these conditionals are very confusing. Rewrite by extracting a
-      //       method or simplify.
-      if (aNodes.length < 3) {
-        // todo: remove nonstandard Element.innerText usage
+  const thArray = []
+  const headers = Polyfill.querySelectorAll(element, 'th')
```

Comment:
According to the JSDocs, this always returns a nonnull array. Please remove the conditional below or update the JSDocs if they're wrong. I believe you could also write this as `headers.forEach(header => { ... })` since the index isn't used elsewhere.

Reply:
Ya I initially used `forEach` but it doesn't support 'break' for early exit.

I'll remove the conditional - good catch!

Reply:
Conditional removed!